LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 2-Clause License
 
-Copyright (c) 2019 Linaro
+Copyright (c) 2020 TrustedFirmware.org
```

Comment:
Should you really replace the previous copyright, or just add the new one?

Reply:
Since I'm representing Linaro on this matter and also are on the TF TSC/Board, I believe this is OK. If there would have been other companies involved here, then I'd definitely keep them.

Reply:
OK fine